### PR TITLE
Fixes related to the renaming from hash to key/checksum

### PR DESCRIPTION
**Pull request by AurelienGasser**
This PR addresses the changes from https://github.com/SubstraFoundation/substra-backend/pull/342 and https://github.com/SubstraFoundation/substra-backend/pull/345

### CODE REVIEW

**Comment by AurelienGasser** [comment]
@jmorel I've clicked around and it seems to generally work, yay!

The only bug I found seems to be unrelated to this PR (but maybe I'm wrong?)

First I click on an Algo from an Objective "Leaderboard"

image

I reach this page, all good (notice the filters at the top)

image

Now if I click the "Objective" menu I get an error

image

image


My understanding is that this issue was already present before the changes from https://github.com/SubstraFoundation/substra-backend/pull/342 and https://github.com/SubstraFoundation/substra-backend/pull/345. Is that correct?

**Comment by jmorel** [comment]
You're absolutely right, it's not related to this PR but rather to the way these global filters were designed.

**Review by jmorel — APPROVED** [review]

**Comment by AurelienGasser** [comment]
@jmorel I can't merge this (the merge button doesn't appear in the UI). Feel free to merge it :)

**Pull request merged**
